Security/Authenticator/JWTAuthenticator.php

```diff
@@ -240,8 +240,8 @@ protected function loadUser(array $payload, string $identity): UserInterface
 
     public function createAuthenticatedToken(PassportInterface $passport, string $firewallName): TokenInterface
```

Comment:
>  (*PassportInterface are deprecated in Symfony 5.4)

What about this argument type here?

Reply:
I think @chalasr was talking about `UserPassportInterface` here, and not the generic `PassportInterface` (the * before makes me say this)

Reply:
I think it is PassportInterface? You can check from here https://github.com/symfony/symfony/blob/5.4/CHANGELOG-5.4.md

Reply:
Of what I've seen the whole `createAuthenticatedToken` is going to be deprecated in favor of the simple `createToken` method in any case :)

Reply:
This method must remain the same for compatibility. Both `PassportInterface` and `createAuthenticatedToken()` are deprecated and removed in Symfony 6.0. 
Note that the reasoning for deprecating `PassportInterface` is that passports are not an extension point, one should use custom badges instead.